### PR TITLE
wsdd2: fix infinite loop when BI_PARM is never set

### PR DESCRIPTION
**Pull request by wongsyrone**
Maintainer:  @Andy2244 
Compile tested: x86_64
Run tested: x86_64

Description:
if BI_PARM is never set, it's `""` and causes infinite loop (before my PR sent upstream) or error out (after my PR sent upstream). Append `-b` option only if it's valid.

PS: should bump the source code revision and release code, this PR doesn't update anything, thus not bump the release code.

### CODE REVIEW

**Comment by Andy2244** [comment]
looks fine thanks

**Comment by wongsyrone** [comment]
@neheb Please review.

**Comment by neheb** [comment]
Needs a PKG_RELEASE bump.

File: net/wsdd2/files/wsdd2.init [hunk 1]
```diff
@@ -68,7 +68,7 @@ start_service() {
 	[ -n "$ifname" ] && procd_append_param command -i "$ifname"
 	procd_append_param command -N "$NB_PARM"
 	procd_append_param command -G "$WG_PARM"
-	procd_append_param command -b "$BI_PARM"
+	[ "x${BI_PARM}" != "x" ] && procd_append_param command -b "$BI_PARM"
```
Comment (neheb) [inline net/wsdd2/files/wsdd2.init:71]: Instead of != &&, could use = ||

**Comment by wongsyrone** [comment]
@neheb Fixed.

**Pull request merged**